toolbox/githubctl/main.go

```diff
@@ -24,14 +24,15 @@ import (
 )
 
 var (
-	owner       = flag.String("owner", "istio", "Github owner or org")
-	tokenFile   = flag.String("token_file", "", "File containing Github API Access Token")
-	op          = flag.String("op", "", "Operation to be performed")
-	repo        = flag.String("repo", "", "Repository to which op is applied")
-	baseBranch  = flag.String("base_branch", "", "Branch to which op is applied")
-	refSHA      = flag.String("ref_sha", "", "Reference commit SHA used to update base branch")
-	nextRelease = flag.String("next_release", "", "Tag of the next release")
-	githubClnt  *u.GithubClient
+	owner                  = flag.String("owner", "istio", "Github owner or org")
+	tokenFile              = flag.String("token_file", "", "File containing Github API Access Token")
+	op                     = flag.String("op", "", "Operation to be performed")
+	repo                   = flag.String("repo", "", "Repository to which op is applied")
+	baseBranch             = flag.String("base_branch", "", "Branch to which op is applied")
+	refSHA                 = flag.String("ref_sha", "", "Reference commit SHA used to update base branch")
+	nextRelease            = flag.String("next_release", "", "Tag of the next release")
+	skipEditDownloadScript = flag.Bool("skip_edit_download_script", false, "Have download script point to old version")
```

Comment:
I'd make the default to not edit that script.

The idea there is you make the release first - then spend some days making sure it's good/better than the previous one and ready for prime time - then update the download script

so I see that update as very rare - also that file is special as in there is only 1 so while once we cut the 0.2 branch, we will change istio.RELEASE in the branch, the download change is on master,...

Reply:
Got it. Thanks for the detailed explanation!